src/discordas.ts

```diff
@@ -214,20 +216,67 @@ async function run(): Promise<void> {
 
     roomhandler.bindThirdparty();
 
+    try {
+        await startDiscordBot(discordbot, appservice.botClient, config.bridge.adminMxid);
+        log.info("Discordbot started successfully");
+    } catch (err) {
+        log.error(err);
+        log.error("Failure during startup. Exiting");
+        process.exit(1);
+    }
+
     await appservice.begin();
     log.info(`Started listening on port ${port}`);
 
+}
+
+interface AdminRooms {
+    [mxid: string]: string;
+}
+
+async function notifyBridgeAdmin(client: MatrixClient, adminMxid: string, message: string) {
+    log.verbose(`Looking up admin room for ${adminMxid}`);
+    let adminRooms = await client.getAccountData<AdminRooms>(ADMIN_ROOMS_KEY).catch(() => null) || {};
+    if (!adminRooms[adminMxid]) {
+        log.verbose(`Creating an admin room for ${adminMxid}`);
+        adminRooms[adminMxid] = await client.createRoom();
+        await client.inviteUser(adminMxid, adminRooms[adminMxid]);
```

Comment:
Any danger in inviting the users as part of the createRoom opts?

Reply:
No, I just haven't dug up the docs for that ;) Will give it a shot, might save me some work later on.

Reply:
Kept in as-is since it makes cleanup possible in case the invite didn't work for some reason.